ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/services/MeteringReportServiceAuthorization.scala

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
let’s please implement the Scala side in a separate PR.

Reply:
Will do.

Reply:
Done, and now merged.